.kitchen.yml

```diff
@@ -1,6 +1,8 @@
 ---
 driver:
   name: vagrant
+  vagrantfiles: 
+      - vagrantfile.rb
```

Comment:
Happy if I kill the other but keep the .rb one named as is? If l capitalise it rubocop will complain, if I drop the .rb test kitchen will complain :)

Sent from my iPhone

> On 20 Apr 2015, at 20:25, Nathan Lee <notifications@github.com> wrote:
> 
> In .kitchen.yml:
> 
> > @@ -1,6 +1,8 @@
> >  ---
> >  driver:
> >    name: vagrant
> > +  vagrantfiles: 
> > +      - vagrantfile.rb
> I don't think the other vagrantfile is needed. I would just overwrite it.
> 
> —
> Reply to this email directly or view it on GitHub.
>